src/core.js

```diff
@@ -186,7 +186,7 @@ $.extend( $.fn, {
 } );
 
 // Custom selectors
-$.extend( $.expr[ ":" ], {
+$.extend( $.expr.pseudos || $.expr[ ":" ], {
```

Comment:
Please add a comment that we have a jq 1.7 bc related line here, so we dont miss it when we drop 1.7.x support